diff/diff_test.go

```diff
@@ -232,8 +238,18 @@ func TestParseFileDiffHeaders(t *testing.T) {
 			t.Fatalf("%s: got ParseFileDiff error %v", test.filename, err)
 		}
 
+		got, want := diff, test.wantDiff
+		// reflect.DeepEqual will compare OrigTime and NewTime by pointers
```

Comment:
instead of this custom manual comparison I recommend using https://godoc.org/github.com/google/go-cmp/cmp instead of `reflect.DeepEqual`.